src/vma/dev/ring_tap.h

```diff
@@ -112,25 +112,29 @@ class ring_tap : public ring_slave
 	inline void set_vf_ring(ring_slave *p_ring) { m_vf_ring = p_ring; }
 	inline void inc_vf_plugouts() { m_p_ring_stat->tap.n_vf_plugouts++; }
 	inline ring_slave* get_vf_ring() { return m_vf_ring; }
+
+	inline int get_tap_if_index() { return m_tap_if_index; }
+	inline int get_tap_fd() { return m_tap_fd; }
 protected:
 
 private:
 	inline void return_to_global_pool();
-	void prepare_flow_message(vma_msg_flow& data,
-			flow_tuple& flow_spec_5t, msg_flow_t flow_action);
+	void prepare_flow_message(vma_msg_flow& data, msg_flow_t flow_action,
+			flow_tuple& flow_spec_5t);
+	void prepare_flow_message(vma_msg_flow& data, msg_flow_t flow_action);
```

Comment:
Daemon is not aware that this new message type has no dst ip/port... you are sending garbage.